sdk/python/core/ydk/services/crud_service.py

```diff
@@ -206,7 +206,7 @@ def _create_update_entity_filter(self, entity):
             raise YPYServiceError(error_msg=err_msg)
 
         module = importlib.import_module(entity._meta_info().pmodule_name)
-        update_filter = getattr(module, entity._meta_info().name)()
+        update_filter = operator.attrgetter(entity._meta_info().name)(module)()
```

Comment:
Thanks for your contribution. Need to import the `operator` module. This is causing the [CI to fail](https://travis-ci.org/CiscoDevNet/ydk-gen/jobs/246093661#L1974).